packages/ui/src/components/va-switch/VaSwitch.vue

```diff
@@ -154,16 +159,19 @@ export default defineComponent({
       return props.label
     })
 
-    const computedClass = computed(() => ({
-      'va-switch--checked': isChecked.value,
-      'va-switch--indeterminate': isIndeterminate.value,
-      'va-switch--small': props.size === 'small',
-      'va-switch--large': props.size === 'large',
-      'va-switch--disabled': props.disabled,
-      'va-switch--readonly': props.readonly,
-      'va-switch--left-label': props.leftLabel,
-      'va-switch--error': computedError.value,
-      'va-switch--on-keyboard-focus': hasKeyboardFocus.value,
+    const computedClass = useBem('va-switch', () => ({
+      ...pick(props, ['readonly', 'disabled']),
+      checked: isChecked.value,
+      indeterminate: isIndeterminate.value,
+      small: props.size === 'small',
+      large: props.size === 'large',
+      'left-label': props.leftLabel,
+      error: computedError.value,
+      'on-keyboard-focus': hasKeyboardFocus.value,
```

Comment:
we need a small update to `useBem` (in a separate release) to be able to write in camelCase

Reply:
Yes, i agree, will create separate issue.